pipeline/values.yaml

```
@@ -7,6 +7,8 @@ global:
     clientid: ""
     clientsecret: ""
 
+signupredirectpath: /ui
```

```suggestion
signupredirectpath: /ui/
```

to prevent one more redirect